cmd/fetch_test.go

```diff
@@ -2,77 +2,157 @@ package cmd
 
 import (
 	"context"
+	"fmt"
+	"net/http"
+	"regexp"
 	"testing"
 	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/qri-io/qri/api"
+	"github.com/qri-io/qri/config"
+	"github.com/qri-io/qri/lib"
+	"github.com/qri-io/qri/logbook"
 )
 
 func TestFetchCommand(t *testing.T) {
-	r := NewTestRunner(t, "peer_a", "qri_test_fetch_a")
-	defer r.Delete()
+	a := NewTestRunner(t, "peer_a", "qri_test_fetch_a")
+	defer a.Delete()
 
-	ctx, done := context.WithCancel(context.Background())
-	defer done()
+	// TODO(dustmop): Move most of the below hooks into a common testRunner. Maybe the basic
+	// TestRunner will work?
```

Comment:
The test runner itself looks like it's worth a little scrutiny, but agree with putting a TODO here & bundling up examining the test runner for another day